ui/apps/platform/src/Containers/VulnMgmt/List/Cves/VulnMgmtListCves.js

```diff
@@ -376,8 +385,11 @@ const VulnMgmtCves = ({
         e.stopPropagation();
 
         const currentEntityType = workflowState.getCurrentEntity().entityType;
-        const cvesToToggle = cve ? [cve] : selectedCveIds;
-        suppressVulns(cveType, cvesToToggle, duration)
+        const cveIdsToToggle = cve ? [cve] : selectedCveIds;
```

Comment:
Something I noticed when I added a `console.log` to see examples of `#` in id.

Does `addToPolicy` need internal id or external name? Expand to see unchanged code above.

Reply:
^ @md2119 Related to this bug, but Mark asked a good question, about whether a policy created to detect specific CVEs can take the ID, rather than the name.

Reply:
Policies work with external CVE name

Reply:
> Policies work with external CVE name

I found that we were picking off just the CVEs already for Add to Policy, but there was a bug where selecting multiple IDs that started with the same name resulting in multiple policy criteria for the same CVE name.

I refactored everything to use the same helper function, cleaned up some pre-Postgres code I found lying around, and added a few tests.

Bug I fixed:
<img width="1503" alt="Screen Shot 2023-09-12 at 6 34 41 PM" src="https://github.com/stackrox/stackrox/assets/715729/48704f80-e2b7-4527-b3a2-bb700ef02268">